### PR TITLE
docs(tutorial): Use mimetypes from standard lib

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: doc/user/tutorial.rst [hunk 1]
```diff
@@ -399,9 +373,7 @@ Go ahead and edit your ``images.py`` file to look something like this:
             self.storage_path = storage_path
 
         def on_get(self, req, resp, name):
-            ext = os.path.splitext(name)[1][1:]
-            resp.content_type = _ext_to_media_type(ext)
-
+            resp.content_type = mimetypes.guess_type(name)[0]
```
Comment (davidbgk) [inline doc/user/tutorial.rst]: `guess_type` always [returns a tuple](https://docs.python.org/2/library/mimetypes.html#mimetypes.guess_type).

File: doc/user/tutorial.rst [hunk 2]
```diff
@@ -350,35 +339,20 @@ Go ahead and edit your ``images.py`` file to look something like this:
...
         def __init__(self, storage_path):
             self.storage_path = storage_path
 
         def on_post(self, req, resp):
-            image_id = _generate_id()
-            ext = _media_type_to_ext(req.content_type)
-            filename = image_id + '.' + ext
-
+            ext = mimetypes.guess_extension(req.content_type)
+            filename = '{uuid}{ext}'.format(uuid=uuid.uuid4(), ext=ext)
```
Comment (davidbgk) [inline doc/user/tutorial.rst]: `guess_extension` always [includes the leading dot](https://docs.python.org/2/library/mimetypes.html#mimetypes.guess_extension).

**Comment by kgriffs** [comment]
Thanks, this looks great! I'm assuming you tested this manually? (I'm reminded that at some point we need to find a way to automatically test samples from the tutorial. :stuck_out_tongue:)

**Comment by davidbgk** [comment]
@kgriffs yes, I tested it because I need it for my app :) Testing the documentation is always a hard problem…

**Comment by kgriffs** [comment]
Cool, LGTM.

**Pull request merged**